src/job_description_feature_extraction.py

```diff
@@ -0,0 +1,49 @@
+import string
+import operator
+from collections import Counter
+
+
+def get_one_hot_encoded_words(feature_to_extract):
+    """
+    get_feature_word_count(x) returns a list of tuples which consists of word:count
+    From those lists, the words appear in the list below ( word_list ) was hand picked
+    Fhey are the word which might have an impact on the salary
+    """
+    word_list = ['excellent', 'graduate', 'immediate', 'junior', 'urgent']
```

Comment:
I extracted all words that occurred more than 0.5% of the data(per row), something like 50~ returned. Out of those 50, these were the only ones that seemed relevant(might affect the salary), hence only those words. Automating this process is not feasible for our task because this will be done only once.